doc/lua_api.txt

```diff
@@ -567,6 +567,19 @@ Example:
 Creates an inventorycube with `grass.png`, `dirt.png^grass_side.png` and
 `dirt.png^grass_side.png` textures
 
+#### `[rect:<w>x<h>:<color>`
```

Comment:
Fill is a much better name.

What would the expected behaviour of a position offset be when the source image is smaller than the rect?
What would the expected image-scaling behaviour be if the position offset is negative?

Reply:
I was doing some work with Image Magick last night, it has its own image manipulation language and uses 'draw rectangle'. This had me thinking `[drawrect` would also be better than `[rect`, and you could say "^[draw... functions don't scale the base image". But since there'll probably never be anything like [drawline or [drawcirc I'm happy to keep it simple with [fill

Reply:
renamed to fill in 8252840
optional position offset added in e211ce7